external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -229,7 +229,7 @@ impl<Progress: Write> DependencyGraphBuilder<Progress> {
         // part of the newly computed dependency graph
         let new_manifest_digest = digest_str(manifest_string.into_bytes().as_slice());
         let lock_path = root_path.join(SourcePackageLayout::Lock.path());
-        let lock_file = File::open(lock_path);
+        let lock_file = File::open(lock_path.clone());
```

Comment:
nit: should be able to open a file by passing a reference to the lock path.

```suggestion
        let lock_file = File::open(&lock_path);
```